### PR TITLE
add clang-format

### PR DESCRIPTION
**Pull request by mat-hek**

### CODE REVIEW

File: focal/Dockerfile [hunk 1]
```diff
@@ -6,30 +6,31 @@ SHELL ["/bin/bash", "-c"]
...
+  libreadline-dev \
+  libssl-dev \
+  libtool \
+  libxslt-dev \
+  libyaml-dev \
+  locales \
+  unixodbc-dev \
+  unzip \
+  wget \
+  clang-format \
```
Comment (mkaput) [inline focal/Dockerfile:30]: 1) seems like formatting got messed up
2) please keep this list sorted :)

Comment (mat-hek) [reply focal/Dockerfile]: 1. vscode formats like that, if there are any specific formatting rules let's specify them somewhere

Comment (mkaput) [reply focal/Dockerfile]: vscode has very dumb formatting for Dockerfiles, I recommend manual formatting :)

Comment (mat-hek) [reply focal/Dockerfile]: done

**Pull request merged**